CHANGELOG.md

```diff
@@ -21,6 +20,12 @@
 
 - Feature: The agent injector now supports a new annotation, `telepresence.getambassador.io/inject-ignore-volume-mounts`, that can be used to make the injector ignore specified volume mounts denoted by a comma-separated string.
 
+- Change: The traffic manager is no longer automatically installed into the cluster. Connecting or creating an intercept in a cluster without a traffic manager will return an error.
+
+- Change: The traffic manager can be installed via `telepresence helm install`.
```

Comment:
I'd consider this a feature and I think that it applies to the `helm` command as a whole, so the text should be something like "Feature: A new `telepresence helm` command was added to provide an easy way to install, upgrade, or uninstall the telepresence traffic-manager."